src/xss-filters.js

```diff
@@ -43,26 +47,164 @@ exports._getPrivFilters = function () {
+                            : num === 0x91 ? '\u2018'  // LEFT SINGLE QUOTATION MARK (‘)
+                            : num === 0x92 ? '\u2019'  // RIGHT SINGLE QUOTATION MARK (’)
+                            : num === 0x93 ? '\u201C'  // LEFT DOUBLE QUOTATION MARK (“)
+                            : num === 0x94 ? '\u201D'  // RIGHT DOUBLE QUOTATION MARK (”)
+                            : num === 0x95 ? '\u2022'  // BULLET (•)
+                            : num === 0x96 ? '\u2013'  // EN DASH (–)
+                            : num === 0x97 ? '\u2014'  // EM DASH (—)
+                            : num === 0x98 ? '\u02DC'  // SMALL TILDE (˜)
+                            : num === 0x99 ? '\u2122'  // TRADE MARK SIGN (™)
+                            : num === 0x9A ? '\u0161'  // LATIN SMALL LETTER S WITH CARON (š)
+                            : num === 0x9B ? '\u203A'  // SINGLE RIGHT-POINTING ANGLE QUOTATION MARK (›)
+                            : num === 0x9C ? '\u0153'  // LATIN SMALL LIGATURE OE (œ)
+                            : num === 0x9E ? '\u017E'  // LATIN SMALL LETTER Z WITH CARON (ž)
+                            : num === 0x9F ? '\u0178'  // LATIN CAPITAL LETTER Y WITH DIAERESIS (Ÿ)
+                            : (num >= 0xD800 && num <= 0xDFFF) || num === 0x0D ? '\uFFFD'
+                            : x.frCoPt(num);
+                }
+                return namedRefMap[named || named1] || m;
+            });
+            return callback ? callback.apply(decodedStr, args) : decodedStr;
+        });
+    }
+
+    function cssEncode(chr) {
+        // space after \\HEX is needed by spec
+        return '\\' + chr.charCodeAt(0).toString(16).toLowerCase() + ' ';
+    }
+    function css(s, reSensitiveChars) {
+        return htmlDecode(s, null, null, function() {
+            return this.replace(reSensitiveChars, cssEncode);
+        });
+    }
+    function cssUrl(s, reSensitiveChars) {
+        return htmlDecode(s, null, null, function() {
+            // encodeURI() will throw error for use of the CSS_UNSUPPORTED_CODE_POINT (i.e., [\uD800-\uDFFF])
+            var s = x.yufull(this), protocol = getProtocol(s);
+            // prefix ## for blacklisted protocols
+            s = protocol && URI_BLACKLIST_PROTOCOLS[protocol.toLowerCase()] ? '##' + s : s;
+
+            return reSensitiveChars ? s.replace(reSensitiveChars, cssEncode) : s;
+        });
+    }
 
     return (x = {
+        // turn invalid codePoints and that of non-characters to \uFFFD, and then fromCodePoint()
+        frCoPt: function(num) {
+            return !isFinite(num) ||            // `NaN`, `+Infinity`, or `-Infinity`
+                num <= 0 ||                     // NULL or not a valid Unicode code point
+                num > 0x10FFFF ||               // not a valid Unicode code point
+                // Math.floor(num) != num || 
+
+                (num >= 0x01 && num <= 0x08) ||
```

Comment:
it's a simplified fromCodePoint():
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/fromCodePoint#Polyfill